### PR TITLE
update source_version and base_version

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by guilhermeleobas** [comment]
@conda-forge-admin, please rerender

**Comment by guilhermeleobas** [comment]
@conda-forge-admin, please restart ci

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,9 +23,9 @@ package:
 
 source:
   url: https://github.com/omnisci/omniscidb/archive/v{{ source_version }}.tar.gz
-  sha256: a41099704792cf928c9bc8e7f1c6ede1a400b0c024a8d22e8c4daaff329bd451
+  sha256: c800f768c52a06bc6c1f9140cae427becc3adea0bd3947b80cc703afdeef91b8
   patches:
-    - 0001-Fix-DBE-build.patch
+    - 0001-Fix-DBE-build.patch  # [version == "5.10.1"]
```
Comment (pearu) [inline recipe/meta.yaml:28]: Since we don't build for 5.10.1 anymore, this patch as well as the patch file itself can be removed:
```suggestion
```

**Review by pearu — CHANGES_REQUESTED** [review]
We don't need the 5.10.1 specific patch file.

**Pull request closed without merging**